### PR TITLE
fix: initalize routes of list pages

### PR DESCRIPTION
**Pull request by hmble**
Fix for issue #181 

I don't know how to test is this right or not. 
I will update this PR if any changes are needed.

### CODE REVIEW

**Comment by dominikbraun** [comment]
Sorry, I managed to write `Route: ...` in my example. My bad.

`Route` already is included inside `Page`, so the syntax is:

```go
&model.ListPage{
	Page: model.Page{
		Route: tagsDir + "/" + key,
	},
	Pages: make([]*model.Page, 0),
}
```

File: model/page.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func (p *Page) SetProvidedType(providedType string) {
 type ListPage struct {
 	Page
 	Pages []*Page
+	Route string
```
Comment (dominikbraun) [inline model/page.go:51]: `Route` can be removed since it already exists in `Page`.

File: plugin/tags/tags.go [hunk 2]
```diff
@@ -77,5 +77,6 @@ func (t *tags) PostWrite() error {
 func (t *tags) createListPage(key string) {
 	t.m[key] = &model.ListPage{
 		Pages: make([]*model.Page, 0),
+		Route: tagsDir + "/" + key,
```
Comment (dominikbraun) [inline plugin/tags/tags.go:80]: Therefore, this becomes

```
Page: model.Page{
	Route: tagsDir + "/" + key,
},
```

**Review by dominikbraun — CHANGES_REQUESTED** [review]
As I mentioned, I made a mistake when describing the solution in #180. 😅

**Review by dominikbraun — APPROVED** [review]
LGTM 👍

**Comment by hmble** [comment]
Hey @dominikbraun I have a request which I sent you on your mail at ( mail at dominikibraun.io ) 
Can you please check that for me ? It will help me a lot.

**Pull request merged**